### PR TITLE
Replace "world" with "anyone" (test)

### PR DESCRIPTION
**Pull request by For124**

### CODE REVIEW

**Comment by Luapulu** [comment]
Nice PR!

**Comment by For124** [comment]
Hallo

File: src/main.rs [hunk 1]
```diff
@@ -1,3 +1,3 @@
 fn main() {
-    println!("Hello, world!");
+    println!("Hello, anyone!");
```
Comment (Luapulu) [inline src/main.rs]: Writing anyone is a nice touch. I think everyone would be more inclusive though.

Comment (For124) [reply src/main.rs]: True True

**Review by Luapulu — APPROVED** [review]
Perfect!

**Pull request closed without merging**